source/vkCodes.py

```diff
@@ -1,8 +1,8 @@
-#vkCodes.py
-#A part of NonVisual Desktop Access (NVDA)
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
-#Copyright (C) 2007-2010 Michael Curran <mick@kulgan.net>, James Teh <jamie@jantrid.net>
+# vkCodes.py

```

Comment:
This line, and all lines which just contains the file name should be removed according to the copyright header standards on the Wiki. Also it feels a bit weird  to me to modify the file just to reformat its copyright header so this should probably be reverted.